docs/ngrok-agent/config.md

```diff
@@ -392,4 +392,4 @@ Below is an example configuration file with all the options filled in.
           - region=us-east
           - team=infra
         addr: 8000
-        inspect: false
\ No newline at end of file
+        inspect: false
```

Comment:
Looks like the GitHub editor removed a blank space from the end of this line. Let me know if that's an issue. 😅